ledger/eval.go

```diff
@@ -963,6 +991,7 @@ func eval(ctx context.Context, l ledgerForEvaluator, blk bookkeeping.Block, vali
 	if err != nil {
 		return StateDelta{}, err
 	}
+	go prefetchThread(ctx, eval.state.lookupParent, blk.Payset)
```

Comment:
The `ctx` here might not be the best context to use. I think it would be better to copy the 
```
validationCtx, validationCancel := context.WithCancel(ctx)
defer validationCancel()
```
from few lines below and use it here. That way, whenever this function exits OR the calling context is canceled, the prefetchThread would abort.